sherpa/utils/err.py

```diff
@@ -313,7 +313,7 @@ class ModelErr(SherpaErr):
     dict = {'numthawed': "expected %d thawed parameters, got %d",
             'badinstance': '%s instance cannot be created from another model',
             'noncall': 'attempted to create %s from non-callable object of type %s',
-            'needsint': 'An integrated grid is required for model evaluation,\ne.g. [0.1,0.2],[0.2,0.3]',
+            'needsint': 'A non-overlap integrated grid is required for model evaluation,\ne.g. [0.1,0.2],[0.2,0.3]',
```

Comment:
I think it reads better with 'A non-overlapping '

Reply:
Done